packages/build-info/src/build-info.ts

```diff
@@ -0,0 +1,30 @@
+export class BuildInfo {
```

Comment:
Is this used?

Reply:
sorry this PR is highly draft (forgot to convert it to draft mode)